perf_dashboard/perf_dashboard/settings.py

```diff
@@ -38,10 +38,10 @@
 SECRET_KEY = '0-k61x@nbmohu87^f!gql9ug1c*5(8k@ujeeme%cx4x=d@*=-f'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
? shouldn't we have this false

Reply:
yes< should be false, that is for my local dev, i pushed that file, should be removed from the commit.